src/ensembl/src/shared/accordion/components/Accordion.tsx

```diff
@@ -12,28 +13,40 @@ type AccordionProps = Pick<
   preExpanded?: UUID[];
   allowMultipleExpanded: boolean;
   allowZeroExpanded: boolean;
+  className?: any;
```

Comment:
Not a string?

Reply:
I guess `className` is not required as it'll be in the `DivAttributes` anyways.